Project/demo/src/main/java/com/example/demo/controller/HelloController.java

```diff
@@ -0,0 +1,12 @@
+package com.example.demo.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
```

Comment:
Please fix the formatting, in Armedia projects we're using this code style:
BAD CODE FORMATTING: 
```
@RestController
public class HelloController {
    @RequestMapping("/")
    String index() {
        return "Hello, World!";
    }
} 
```

GOOD CODE FORMATTING:
```
@RestController
public class HelloController 
{
    @RequestMapping("/")
    String index() 
    {
        return "Hello, World!";
    }
} 
```